BOUNTY.md

```diff
@@ -1,24 +1,33 @@
+[![license](https://img.shields.io/github/license/musicoin/desktop.svg?style=for-the-badge)](https://github.com/Musicoin/desktop/blob/master/LICENSE)
+![GitHub top language](https://img.shields.io/github/languages/top/musicoin/desktop.svg?style=for-the-badge)
+[![GitHub Release Date](https://img.shields.io/github/release-date/Musicoin/desktop.svg?style=for-the-badge)](https://github.com/Musicoin/desktop/releases) 
+[![Github All Releases](https://img.shields.io/github/downloads/Musicoin/desktop/total.svg?style=for-the-badge)](https://github.com/Musicoin/desktop/releases) 
+[![GitHub issues](https://img.shields.io/github/issues-raw/badges/shields.svg?style=for-the-badge)](https://github.com/Musicoin/desktop/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) 
+![GitHub contributors](https://img.shields.io/github/contributors/Musicoin/desktop.svg?style=for-the-badge) 
+
+
+---
+
 # The Musicoin Bounty Program
 
-## About the Program
-Musicoin Desktop Wallet(MDW) is an open source project to support end users to well transfer the currency($MUSIC) between different stops. It's been downloaded more than 40K times since it's first release on Feb. 2017, a big proof that it's a useful and fast-growign tool.
+Musicoin ($MUSIC) Desktop Wallet(MDW) is an open source project to support end users to transfer $MUSIC between wallets. So far, the project is drumming up quite a bit of interest. So help out! (Pun intended.)
 
-Unfortunately, the focus of core team has to be more on blockchain upgrading, web application development, and mobile/hardware partnership, to make the ecosystem bigger and widely adopted by various stakeholders. This has shifted our focus from MDW to other applications, leading us to introduce this bounty program.
+The core development team is tasked with upgrading and improving the blockchain itself, the web app, and a mobile app. This has shifted the focus of the project away from the MDW, and toward other applications more targeted to widening the spread of adoption and community in the ecosystem.
 
-## Guidelines and Instructions
+And here is where you come in! We have created a bounty program for you to help out with your own contributions. If you want to hack on this codebase, read on!! We appreciate your interest in the project, and your effort into the project.
 
-Issues that have a bounty are marked with two different kinds of labels - A Cyan bounty label along with the bounty amount in its respective color. Participants interested in putting in a PR for the same are requested to do so right away, there is no need to claim the issue or request to be assigned towards the same.
+Issues that have a bounty are marked with two labels. Firstly, a cyan bounty label denotes that an issue has a bounty reward associated with it. Secondly, the amount of the reward in $MUSIC. Should you find that one of the bounty issues falls into your wheelhouse; by all means, submit a PR!
```

Comment:
This line looks good, but we removed the line which says that there is no need to claim the issue and people might have doubts. Would be useful to add it back in.

Reply:
I placed it back in, but without "towards the same" - only because it read confusingly in my own mind. It's good to know that you've tested even text files for functionality!